ci-operator/step-registry/openshift/manifests/kubelet-logs-to-console/openshift-manifests-kubelet-logs-to-console-ref.metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+	"path": "openshift/manifests/kubelet-logs-to-console/openshift-manifests-kubelet-logs-to-console-ref.yaml",
+	"owners": {
+		"approvers": [
+			"smarterclayton",
+			"wking",
+			"vrutkovs",
+			"abhinavdahiya",
```

Comment:
No longer with the company fwiw.

Reply:
That file is a symlink from manifests, so we'll need a PR to clean up all instances